samples/matter/lock/Kconfig

```diff
@@ -97,6 +97,14 @@ endif # CHIP_WIFI
 config CHIP_ICD_SUBSCRIPTION_HANDLING
 	default y
 
+config MATTER_LOCK_LTO
```

Comment:
Please do not include this change. It was attempted as a workaround by @awojasinski-nordicsemi to fix the flash overflow, but apparently it didn't help and the flash overflow was fixed by other means.

Reply:
ok, I'll drop this commit

Reply:
Yup, I can confirm that it should be dropped